redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,345 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crwoperator.v1.1.0
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion":"org.eclipse.che/v1",
+          "kind":"CheCluster",
+          "metadata":{
+             "name":"codeready"
+          },
+          "spec":{
+             "server":{
+                "cheFlavor":"codeready",
+                "cheImage": "quay.io/crw/server-container",
+                "cheImageTag": "1.1-45",
+                "tlsSupport":false,
+                "selfSignedCert":false
+             },
+             "database":{
+                "externalDb":false,
+                "chePostgresHostname":"",
+                "chePostgresPort":"",
+                "chePostgresUser":"",
+                "chePostgresPassword":"",
+                "chePostgresDb":""
+             },
+             "auth":{
+                "openShiftoAuth":false,
+                "externalKeycloak":false,
+                "keycloakURL":"",
+                "keycloakRealm":"",
+                "keycloakClientId":""
+             },
+             "storage":{
+                "pvcStrategy":"common",
+                "pvcClaimSize":"1Gi",
+                "preCreateSubPaths": true
+             }
+          }
+        }
+      ]
+    certified: "true"
+    categories: "Developer Tools"
+    containerImage: quay.io/crw/operator-container:1.1-7
+    createdAt: 2019-03-06
```

Comment:
Please check our docs for our supported datetime format: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
Fixed.